addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp

```diff
@@ -0,0 +1,563 @@
+	};
+	scope[] = {
+		"cup_optic_eotech553_black",
+		"cup_optic_compm4",
+		"cup_optic_compm2_low",
+		"cup_optic_compm2_black"
+	};
+	attachment[] = {
+		"cup_acc_anpeq_2_flashlight_black_l",
+		"cup_acc_anpeq_2_black_top"
+	};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_Stanag"),
+		LIST_2("CUP_30Rnd_556x45_Stanag_Tracer_Red"),
+		LIST_2("HandGrenade"),
+		"SmokeShell"
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	primaryWeapon[] = {"CUP_arifle_M16A4_GL"};
+	attachment[] = {"cup_acc_anpeq_2_grey"};
+	vest[] = {"CUP_V_B_Eagle_SPC_GL"};
+	backPack[] = {"CUP_B_AssaultPack_Coyote"};
+	magazines[] += {
+		LIST_16("CUP_1Rnd_HEDP_M203"),
+		LIST_2("1Rnd_Smoke_Grenade_shell")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
+	primaryWeapon[] = {"CUP_arifle_M4A1_black"};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_Stanag"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	vest[] = {"CUP_V_B_Eagle_SPC_Corpsman"};
+	backpack[] = {"CUP_B_USMC_MOLLE_WDL"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"B_Carryall_cbr"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	primaryWeapon[] = {"CUP_smg_MP5A5"};
+	scope[] = {};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_9x19_MP5"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	items[] += {"ACE_MapTools"};
+	vest[] = {
+		"CUP_V_B_Eagle_SPC_TL",
+		"CUP_V_B_MTV_TL"
+	};
+	magazines[] += {
+		LIST_2("CUP_1Rnd_StarFlare_Red_M203"),
+		LIST_2("CUP_1Rnd_SmokeRed_M203")
+	};
+	linkedItems[] += {
+		"ACE_Yardage450",
+		"ItemGPS"
+	};
+};
+
+class sl : ftl {
+	displayName = "Squad Leader";
+	vest[] = {
+		"CUP_V_B_Eagle_SPC_SL",
+		"CUP_V_B_MTV_PistolBlack"
+	};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	magazines[] += {
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"CUP_H_USMC_CAP_PRR_DES"};
+	vest[] = {"CUP_V_B_Eagle_SPC_Officer"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	vest[] = {"CUP_V_B_Eagle_SPC_RTO"};
+	backpack[] = {"usm_pack_alice_prc77"};
+	backpackItems[] = { FAC_GEAR };
+	linkedItems[] += {
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	primaryWeapon[] = {"hlc_m249_pip4"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	vest[] = {
+		"CUP_V_B_Eagle_SPC_AR",
+		"CUP_V_B_MTV_MG"
+	};
+	backpack[] = {"CUP_B_AssaultPack_Coyote"};
+	magazines[] = {
+		LIST_3("hlc_200rnd_556x45_M_SAW"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpack[] = {"B_Kitbag_cbr"};
+	backpackItems[] = {
+		LIST_3("hlc_200rnd_556x45_M_SAW"),
+		"ACE_EntrenchingTool"
+	};
+	linkedItems[] += {"ACE_Yardage450"};
```

Comment:
Yardage for AARs?

Reply:
Yup! Since I don't want to add magnified optics to the loadout (despite being accurate), I decided to give the AAR some goodies to make up for that.